### DISCUSSION TITLE
what i need to do make one like this can i make one like this

### DISCUSSION THREAD

**Discussion by RV2003331** [obsproject/obs-studio#10064] [Development Questions]
![20240103_165246](https://github.com/obsproject/obs-studio/assets/149759881/46a12a74-1bed-4979-8063-9e7e287dc13a)

**Comment by Lain-B** [comment]
Hi there! This is a place for discussions about OBS itself, not for asking for help making software or asking general programming questions. Judging from this post and your other post, we can't really help you with the kind of software you want to make. You will unfortunately have to look elsewhere for help. Good luck though!
